src/LibraryViewExtension/Handlers/NodeItemDataProvider.cs

```diff
@@ -25,18 +25,22 @@ class LoadedTypeItem
         public string keywords { get; set; }
     }
 
-    class LoadedTypeData
+    class LoadedTypeData<T>
```

Comment:
But if you mean LoadedTypeData<A> can not be casted to LoadedTypeData<B> directly, then you are right. But this is the intention here.